migrations/sql/V2021.03.10.09.45__update_now_application_identity_and_notice_of_work_view.sql

```diff
@@ -0,0 +1,68 @@
+/*set all existing applications to notice of work type */
+update now_application_identity
+set application_type_code = 'NOW';
+
+/* set type column as not null*/
+ALTER TABLE now_application_identity ALTER COLUMN application_type_code SET DEFAULT 'NOW'; 
+ALTER TABLE now_application_identity ALTER COLUMN application_type_code SET NOT NULL ;
+
+DROP VIEW IF EXISTS notice_of_work_view;
+CREATE OR REPLACE VIEW notice_of_work_view
+	AS
+SELECT nid.now_application_guid,
+m.mine_guid,
+m.mine_no,
+nid.now_number,
+app.lead_inspector_party_guid,
+concat_ws (' ', p.first_name, p.party_name) AS lead_inspector_name,
+COALESCE(nowt.description, sub.noticeofworktype, msub.noticeofworktype) as notice_of_work_type_description,
+atc.description as application_entity_type_description,
+-- TODO: Map all MMS and vFCBC statuses to their corresponding Core status.
```

Comment:
This TODO could probably be removed from this file and the `R__view_notice_of_work.sql` file.